### PR TITLE
feat(transactions): get last transactions for dashboard

### PR DESCRIPTION
**Pull request by CeebDev**
Linked to ox_core PR : https://github.com/overextended/ox_core/pull/125

image

### CODE REVIEW

File: web/src/layouts/bank/pages/dashboard/components/TransactionItem.tsx
Comment (LukeWasTakenn) [inline web/src/layouts/bank/pages/dashboard/components/TransactionItem.tsx:2]: You're removing components that are in use? 🤨

File: typings/nui.ts
Comment (LukeWasTakenn) [inline typings/nui.ts]: I don't really like the name `reason` for this case, maybe a word like `description` instead?

I'm open to ideas and suggestions.

**Review by LukeWasTakenn — CHANGES_REQUESTED** [review]

File: typings/nui.ts
Comment (CeebDev) [reply typings/nui.ts]: I have in mind the reason of transactions but yeah can be changed to description or message like PEFCL

File: web/src/layouts/bank/pages/dashboard/components/TransactionItem.tsx
Comment (CeebDev) [reply web/src/layouts/bank/pages/dashboard/components/TransactionItem.tsx:2]: It's not used anymore on my version since we can't really rely on a user, i display the reason instead and no need of the avatar component

File: typings/nui.ts
Comment (LukeWasTakenn) [reply typings/nui.ts]: message sounds good 👍

Comment (CeebDev) [reply typings/nui.ts]: Go for message

File: web/src/layouts/bank/pages/dashboard/components/TransactionItem.tsx
Comment (LukeWasTakenn) [reply web/src/layouts/bank/pages/dashboard/components/TransactionItem.tsx:2]: It's used here: <https://github.com/overextended/ox_banking/pull/7/files#diff-a3504dea90e36b50bc1eb22195f37c4132a1ac55413ccf40f4e6ec6eb483de08R16-R18>

But yeah, instead of doing names we should do an icon depending if the transaction is inbound or outbound.

Comment (CeebDev) [reply web/src/layouts/bank/pages/dashboard/components/TransactionItem.tsx:2]: idk what happen but this file is the old version, will update it directly

Comment (CeebDev) [reply web/src/layouts/bank/pages/dashboard/components/TransactionItem.tsx:2]: I have updated the file, i make the avatar thing for inbound / outbound inside this PR ?

Comment (LukeWasTakenn) [reply web/src/layouts/bank/pages/dashboard/components/TransactionItem.tsx:2]: yeah feel free to

Comment (CeebDev) [reply web/src/layouts/bank/pages/dashboard/components/TransactionItem.tsx:2]: Done 
image

**Review by LukeWasTakenn — APPROVED** [review]
LGTM, pending https://github.com/overextended/ox_core/pull/125 from ox_core before I can merge this @thelindat

**Comment by CeebDev** [comment]
Modified no message handling for transactions due to changes in ox_core, we are not including default message in db anymore

**Pull request merged**
